data/json/recipes/recipe_weapon.json

```diff
@@ -240,7 +240,7 @@
     "reversible": true,
     "autolearn": true,
     "qualities": [ { "id": "CUT", "level": 1 }, { "id": "HAMMER", "level": 1 } ],
-    "components": [ [ [ "leather", 30 ], [ "rope_superior_short", 2, "LIST" ], [ [ "scrap", 9 ], [ "razor_blade", 9 ] ] ]
+    "components": [ [ [ "leather", 30 ], [ "rope_superior_short", 2, "LIST" ], [ [ "scrap", 9 ], [ "razor_blade", 9 ] ] ] ]
```

Comment:
```suggestion
    "components": [ [ [ "leather", 30 ], [ "rope_superior_short", 2, "LIST" ] ], [ [ "scrap", 9 ], [ "razor_blade", 9 ] ] ]
```

Reply:
I pushed the fix for the nesting error, yet both Travis and Appveyor say that error is still there. Any ideas?